### ISSUE TITLE
Heroku stacks that TF/CM are using are being deprecated [13,3, 4.3]

### ISSUE THREAD

**Issue by shaunxp20** [Iridescent-CM/curiosity-machine#2458]
We need to update the Heroku stacks that TF/CF are using by June 1st 2021.

> This app is using the Heroku-16 stack, which is deprecated:
> https://devcenter.heroku.com/changelog-items/1958
> 
> From May 1st 2021, Heroku-16 will be end-of-life, and apps using it will no longer receive security updates, and be run at your own risk.
> 
> From June 1st 2021, builds will no longer be allowed for Heroku-16 apps.
> 
> Please upgrade to a newer stack as soon as possible:
> https://devcenter.heroku.com/articles/upgrading-to-the-latest-stack
> https://help.heroku.com/0S5P41DC/heroku-16-end-of-life-faq

**Comment by viviancan** [comment]
6/4/21 Update: Attempted to upgrade CM staging via the heroku dashboard. Push was rejected due to python 3.6.5 not being supported

More [info here](https://devcenter.heroku.com/articles/python-support#supported-runtimes)

```
remote:  !     Requested runtime (python-3.6.5) is not available for this stack (heroku-20).        
remote:  !     Aborting.  More info: https://devcenter.heroku.com/articles/python-support        
remote:  !     Push rejected, failed to compile Python app.        
```

Next steps: continue to investigate
Should we....

1. Upgrade python version?
2. Does heroku-18 support our version of python?
3. Is there a workaround?

Additional resources
https://stackoverflow.com/questions/54837820/error-while-pushing-to-heroku-requested-runtime-is-not-available-for-this-stack

**Comment by viviancan** [comment]
Tried a few different things based on heroku docs, but settled on upgrading python to a supported runtime. After upgrading python, another issue popped up. It seems that the version of grunt-sass we were using had another dependency of node-sass that was no longer available online. The build was failing on the npm step with a 404 error, unable to download that node-sass that was required. Updated grunt-sass dependency to fix 404 issue. Updated mandrill dependency from 1.57-->1.59 due to issue with asyc being a reserved word in python 3.7.10.

I think once this is merged to staging, staging server stack will be upgraded. The upgrade is currently pending on the heroku dashboard. 

@shaunxp20 Can you take a look at these updates, when you have a chance (no rush at all 😊)?

Pull request --> #2467
